source/_css/utils/_variables.scss

```diff
@@ -244,6 +244,7 @@ $pagination-height: 60px;
 // Post thumbnail image
 // Width and height of post's thumbnail image
 $post-thumbnail-image-width: 140px;
+$post-thumbnail-image-max-height: 250px;
```

Comment:
let's precise that this property is for bottom thumbnail image: `post-bottom-thumbnail-image-max-height`